container/containerpostgres/postgres.go

```diff
@@ -54,67 +56,75 @@ func (p *Container) DSN() string {
 	return p.dsn
 }
 
-func New(t *testing.T) *Container {
+func New(t *testing.T, port ...int) *Container {
 	t.Helper()
 
-	addr := os.Getenv("POSTGRES_HOST")
-	var postgresContainer testcontainers.Container
-
-	if addr == "" {
```

Comment:
Sometimes testing in our database than we skip with set POSTGRES_HOST, still good to keep that